docs/dev/protocol-extensions.md

```diff
@@ -197,5 +197,18 @@ use that information for every subsequent query.
 If we send the query to the wrong node/shard, we want to send the RESULT 
 message with additional information about the tablet in `custom_payload`:
 
-  - `tablet_replicas` - information about tablet replicas, for every replica there is information about the host and shard.
-  - `token_range` - information about token range for that tablet in format `(first_token, last_token]`.
+  - `tablet_replicas` - information about tablet replicas, for every
+    replica there is information about the host and shard.
+  - `token_range` - information about token range for that tablet in
+    format `(first_token, last_token]`.
```

Comment:
I see that you took over the entire [bytes map]. But that leaves us for no room to expand in the future. Let's reserve `tablets-routing-v1` for the key in the [bytes map] so that we can add additional payloads later, or `tablets-routing-v2` when we want to extend the data. The contents of the value in the [bytes map] can be just a sequence of values. Please use the same notation as the protocol  - [long] [long] [uuid] [int], so it's consistent with the original document.

Note also that the tokens may fit into a power-of-two tablet distribution, but that's not guaranteed.

Reply:
@avikivity I don't understand how I took over the entire [bytes map]. I took two keys in this bytes map and I don't understand why other can't be used in the future. Could you explain that?

Reply:
You didn't really take over the entire space, but now all users must coexist. A new user can't use `token_range` (a fairly generic name).

I guess it's possible to have namespaces - something like tablet-replication-v1/replicas and tablet-replication-v1/range, but it seems nicer to pack everything so there's just one string lookup.

Reply:
I think it's better. It removes the question of what to do if just one key is there and the other is missing.

Also, remember to use the native_protocol_v4 way of specifying things.

Reply:
@avikivity I have made the change